diff --git a/node_modules/@ethercorps/sveltekit-og/.bun-tag-1f96aa8f083e2f67 b/.bun-tag-1f96aa8f083e2f67
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/node_modules/@ethercorps/sveltekit-og/.bun-tag-ab0e6b602f43abff b/.bun-tag-ab0e6b602f43abff
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/node_modules/@ethercorps/sveltekit-og/.bun-tag-b8fc2e8e6b26f1e2 b/.bun-tag-b8fc2e8e6b26f1e2
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/node_modules/@ethercorps/sveltekit-og/.bun-tag-f5acbb2282df3b16 b/.bun-tag-f5acbb2282df3b16
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/node_modules/@ethercorps/sveltekit-og/.bun-tag-ff8fab64f4c72270 b/.bun-tag-ff8fab64f4c72270
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/index.d.ts b/dist/index.d.ts
index cfae85a27f3626e44fea1fb9e7fe4a09026fdffd..c17779c22e7488cc8e93a79a1397e7e5d8e2df57 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,2 +1,2 @@
-export { ImageResponse } from "./api.js";
+export { ImageResponse, og } from "./api.js";
 export type { ImageResponseOptions } from "./types.js";
diff --git a/dist/index.js b/dist/index.js
index 4602ace79499054465f1c9ad3e1aefec579ac855..9181a9fc87bd10aa80021f1dd2e902639f477792 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1 +1 @@
-export { ImageResponse } from "./api.js";
+export { ImageResponse, og } from "./api.js";
