diff --git a/node_modules/@types/xml2js/index.d.ts b/node_modules/@types/xml2js/index.d.ts
index 4efb418..41a957f 100644
--- a/node_modules/@types/xml2js/index.d.ts
+++ b/node_modules/@types/xml2js/index.d.ts
@@ -8,6 +8,7 @@ export function parseString(
     options: ParserOptions,
     callback: (err: Error | null, result: any) => void,
 ): void;
+export function parseStringSync(str: convertableToString, options?: ParserOptions): Promise<any>;
 export function parseStringPromise(str: convertableToString, options?: ParserOptions): Promise<any>;
 
 export const defaults: {
