diff --git a/index.d.ts b/index.d.ts
index 4efb418321d999e66c33020c9915f69f8522ffa9..41a957feb72060b3c219317049a0773aeccb2a75 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -8,6 +8,7 @@ export function parseString(
     options: ParserOptions,
     callback: (err: Error | null, result: any) => void,
 ): void;
+export function parseStringSync(str: convertableToString, options?: ParserOptions): Promise<any>;
 export function parseStringPromise(str: convertableToString, options?: ParserOptions): Promise<any>;
 
 export const defaults: {
