diff --git a/dist/cjs/index.d.ts b/dist/cjs/index.d.ts
index d556ad76e2a23d600e3ee3024bad9d7dd8beccf7..42f66408633299db8c40b69378de3007d7f2d69a 100644
--- a/dist/cjs/index.d.ts
+++ b/dist/cjs/index.d.ts
@@ -23,6 +23,7 @@ export * from './query-builder/case-builder.js';
 export * from './query-builder/json-path-builder.js';
 export * from './query-builder/merge-query-builder.js';
 export * from './query-builder/merge-result.js';
+export * from './query-builder/select-query-builder-expression.js';
 export * from './raw-builder/raw-builder.js';
 export * from './raw-builder/sql.js';
 export * from './query-executor/query-executor.js';
diff --git a/dist/esm/index.d.ts b/dist/esm/index.d.ts
index d556ad76e2a23d600e3ee3024bad9d7dd8beccf7..42f66408633299db8c40b69378de3007d7f2d69a 100644
--- a/dist/esm/index.d.ts
+++ b/dist/esm/index.d.ts
@@ -23,6 +23,7 @@ export * from './query-builder/case-builder.js';
 export * from './query-builder/json-path-builder.js';
 export * from './query-builder/merge-query-builder.js';
 export * from './query-builder/merge-result.js';
+export * from './query-builder/select-query-builder-expression.js';
 export * from './raw-builder/raw-builder.js';
 export * from './raw-builder/sql.js';
 export * from './query-executor/query-executor.js';
diff --git a/dist/esm/index.js b/dist/esm/index.js
index c85949e9080a6e78f1aef5561ed050cb6d924b15..e7ae8702350bfd02b6e3b7f98d360e3867a940a4 100644
--- a/dist/esm/index.js
+++ b/dist/esm/index.js
@@ -24,6 +24,7 @@ export * from './query-builder/case-builder.js';
 export * from './query-builder/json-path-builder.js';
 export * from './query-builder/merge-query-builder.js';
 export * from './query-builder/merge-result.js';
+export * from './query-builder/select-query-builder-expression.js';
 export * from './raw-builder/raw-builder.js';
 export * from './raw-builder/sql.js';
 export * from './query-executor/query-executor.js';
