diff --git a/lib/sharp.js b/lib/sharp.js
index 3428c5fefae5c7fc02f18a6a9026ac6a7df190e4..b67b4af96964dbfc5a9fac8bfa0860fc4777ab90 100644
--- a/lib/sharp.js
+++ b/lib/sharp.js
@@ -10,13 +10,26 @@ const { familySync, versionSync } = require('detect-libc');
 const { runtimePlatformArch, isUnsupportedNodeRuntime, prebuiltPlatforms, minimumLibvipsVersion } = require('./libvips');
 const runtimePlatform = runtimePlatformArch();
 
-const paths = [
+let paths = [
   `../src/build/Release/sharp-${runtimePlatform}.node`,
   '../src/build/Release/sharp-wasm32.node',
   `@img/sharp-${runtimePlatform}/sharp.node`,
   '@img/sharp-wasm32/sharp.node'
 ];
 
+if (Bun.embeddedFiles.length) {
+	const libvipsLib = Bun.embeddedFiles.find((file) => /libvips-cpp.*/g.test(file.name));
+	const sharpLib = Bun.embeddedFiles.find((file) => /sharp-.*\.node/g.test(file.name));
+
+	if (!libvipsLib || !sharpLib) {
+		throw new Error('Necessary embedded files not found');
+	}
+
+	if (global.sharpPath) {
+		paths = [global.sharpPath];
+	}
+}
+
 let sharp;
 const errors = [];
 for (const path of paths) {
