diff --git a/node_modules/@types/string-pixel-width/index.d.ts b/node_modules/@types/string-pixel-width/index.d.ts
index d2484f3..856e329 100644
--- a/node_modules/@types/string-pixel-width/index.d.ts
+++ b/node_modules/@types/string-pixel-width/index.d.ts
@@ -14,6 +14,7 @@ interface Settings {
         | "georgia"
         | "helvetica"
         | "impact"
+        | "inter"
         | "open sans"
         | "quantify"
         | "tahoma"
